File: CourierAmexAPI/CourierAmex/Models/TemplateModel.cs
namespace CourierAmex.Models
{
    public class TemplateModel
    {
        public int CompanyId { get; set; }
        public int Id { get; set; }
        public required string ModuleId { get; set; }
        public int TransactionId { get; set; }
        public int SubTransactionId { get; set; }
        public required string Name { get; set; }
        public required string Status { get; set; }
        public required string ProcessType { get; set; }
        public DateTime? CreatedAt { get; set; }
        public Guid? CreatedBy { get; set; }
        public DateTime? ModifiedAt { get; set; }
        public Guid? ModifiedBy { get; set; }

    }
}

File: CourierAmexClient/src/app/features/accounting/models/template.model.ts

export interface TemplateModel  {
    id: number;
    companyId: number;
    moduleId: string;
    transactionId: number;
    subTransactionId: number;
    name: string;
    status: string;
    processType: string;


}

export const newTemplate = {
    id: 0,
    companyId: 0,
    moduleId: '',
    transactionId: 0,
    subTransactionId: 0,
    name: '',
    status: '',
    processType: '',
};


